todo_app/data/Item.py

```diff
@@ -0,0 +1,80 @@
+#Class Object for Trello Item
+import requests
+import os
+
+trello_key = os.environ.get('TRELLO_KEY')
+trello_token = os.environ.get('TRELLO_TOKEN')
+listtodo = os.environ.get('LISTTODO')
+
+
+class Item:
+
+    def __init__(self, id, status, title,datelastaction ):
+        self.id = id
+        self.status = status
+        self.title = title
+        self.datelastaction = datelastaction
+
+    @property
+    def id(self):
+        return self._id
+
+    @id.setter
+    def id(self, value):
+        self._id = value
+    
+    @property
+    def status(self):
+        return self._status
+
+    @status.setter
+    def status(self, value):
+        self._status = value
+    
+    @property
+    def title(self):
+        return self._title
+
+    @title.setter
+    def title(self, value):
+        self._title = value
+    
+    @property
+    def dateLastAction(self):
+        return self._dateLastAction
+
+    @dateLastAction.setter
+    def id(self, value):
+        self._id = value
```

Comment:
I suggest you delete lines 18 to 48.

In your constructor you use e.g. `id`, but here you change e.g. `_id` - so these actually refer to different fields than you set in your constructor!